packages/cozy-sharing/package.json

```
@@ -52,10 +51,9 @@
     "react-router": "^5.0.1"
   },
   "peerDependencies": {
-    "@material-ui/core": "4",
     "cozy-client": "^34.1.0",
     "cozy-realtime": "^3.11.0",
-    "cozy-ui": "^45.1.0",
+    "cozy-ui": "^77.6.0",
```

```suggestion
    "cozy-ui": ">=77.6.0",
```

(yes others are not right, but hey...)